### PR TITLE
Renamed field "taux" into "rate" in llx_c_tva & llx_c_revenuestamp

### PR DESCRIPTION
**Pull request by aspangaro**

### CODE REVIEW

File: htdocs/fourn/commande/fiche.php [hunk 1]
```diff
@@ -318,7 +318,7 @@
     	else
     	{
     		$ttc = price2num($_POST['price_ttc']);
-    		$ht = $ttc / (1 + ($tauxtva / 100));
+    		$ht = $ttc / (1 + ($ratetva / 100));
```
Comment (fappels) [inline htdocs/fourn/commande/fiche.php]: $ratetva not set (also $tauxtva -> old bug?)

Why not translate tva to vat?

**Comment by aspangaro** [comment]
Hi fappels,

You are right, the function is not set. I don't no if it's a bug

I begin to work on a complete traduction and for the moment, i work by term to limit lenght of pull request, vat will be the next. ;)

**Comment by rdoursenaud** [comment]
Keeping PR small is a good goal but keeping everything meaningful is even better.
I believe, as an example, that tauxtva to vatrate can be considered a single token and should be modified together.
I you want to keep it for another PR, fine, but please avoid senseless terms like ratetva.

My 2 cents…

**Comment by BebZ** [comment]
My 2 cents, it is simple and meaningful both ways : "taux" has surely more occurencies than "tauxtva" but ratetva looks so fun as an frenglishism :)

**Comment by aspangaro** [comment]
I correct the function not set but I pushed a more complete PR tomorrow.

**Pull request closed without merging**